sql/core/src/test/scala/org/apache/spark/sql/sources/v2/DataSourceV2Suite.scala

```diff
@@ -121,17 +121,6 @@ class DataSourceV2Suite extends QueryTest with SharedSQLContext {
     }
   }
 
-  test("unsafe row scan implementation") {
-    Seq(classOf[UnsafeRowDataSourceV2], classOf[JavaUnsafeRowDataSourceV2]).foreach { cls =>
```

Comment:
Why remove unsafe tests?

Reply:
That's a followup of https://github.com/apache/spark/pull/21118. you removed `SupportsScanUnsafeRow` there and then this test becomes meaningless.

Reply:
Ok.